pycontrails/core/met.py

```diff
@@ -593,6 +593,16 @@ class MetDataset(MetBase):
         interval ``[-180, 180]``. Defaults to False.
     copy : bool, optional
         Copy data on construction. Defaults to True.
+    attrs : dict[str, Any], optional
+        Attributes to add to :attr:`data.attrs`. Defaults to None.
+        Generally, pycontrails :class:`Models` may use the following attributes:
+
+        - ``provider``: Name of the data provider (e.g. "ECMWF").
```

Comment:
I wonder if we want to prefix this attrs somehow? e.g. `pycontrails_provider`, `pycontrails_dataset` ... This would namespace the attrs and hopefully make it clear that the distinction is only meaningful in the context of our library?

Reply:
I feel mixed on this. I think requiring a "provider" key here is akin to us requiring an "aircraft_type" in the `Flight.attrs` on certain models. By analogy, renaming "aircraft_type" to "pycontrails_aircraft_type" would be fairly unwieldy ... we're already quite used to "aircraft_type" as a required `Flight.attrs` key. To me, this is similar in our syntax and usage.

Reply:
I agree with you, feel free to leave as is!